src/main/resources/assets/wurst/lang/en_us.json

```diff
@@ -88,6 +88,7 @@
   "description.wurst.hack.navigator": "A searchable GUI that learns your preferences over time.",
   "description.wurst.hack.nobackground": "Removes the dark background behind inventories.",
   "description.wurst.hack.noclip": "Allows you to freely move through blocks.\nA block (e.g. sand) must fall on your head to activate it.\n\n§c§lWARNING:§r You will take damage while moving through blocks!",
+  "description.wurst.hack.nocomcrash": "Lags servers using the nocom exploit.\nDoes not work on Paper servers.",

```

Comment:
If I understand correctly this exploit only works on Spigot servers. If so, the description should mention that.